dlc/src/main/scala/org/bitcoins/dlc/sign/DLCTxSigner.scala

```diff
@@ -188,6 +190,36 @@ case class DLCTxSigner(
     CETSignatures(cetSigs, refundSig)
   }
 
+  /** Creates CET signatures async */
+  def createCETSigsAsync()(implicit
```

Comment:
I'm fine with this for now but this will likely be deleted/rewritten when I've done the sigpoint computation refactor